src/controllers/usersController.js

```diff
@@ -120,5 +120,45 @@ export const signin = async (req, res) => {
     });
   } catch (error) {
     return errorRes(res, 500, 'There was error while signining in');
+    
   }
 };
+
+export const forgotPassword= async  (req, res)=> {
+  try{const email = req.body.email
+ const user=await User.findOne({where:{email: email}})
+
+  if (!user) {
+  return errorRes(res,404,res.__('No user found with that email address.'))
+  }else{
+ await  sendEmail('forgotPassword',{
+      email:user.email,
+      id:user.id
+    })
+    successRes(res,200,res.__("check your email",))
+  }}
+  catch(error){
+    return errorRes(res,500,res.__('error while requesting!'))
+  }
+}
+export const resetPassword= async (req, res)=>{
+
+  try{const{ email,password}=req.body
+  
+  const user=await User.findOne({where:{email:email}})
+  if(!user){
+    return errorRes(res,500,res.__('Sorry!No user found with that email address.'))
+  }else{
+    const newPassword  = await bcrypt.hash(password,10, async(err,result)=>{
+      if(result){
+     await  User.update({password:newPassword},{where:{id:user.id}})
+
+    //  await sendEmail('resetPassword',{email:user.email})
```

Comment:
if this commented line is no longer in use should be removed

Reply:
thank you